bin/oneoff/back_populate_contacts_hashed_email

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+
+# This script populates the hashed_email column of the pegasus contacts table.
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'digest/md5'
+require src_dir 'database'
+
+def main
+  DB[:contacts].each do |contact|
```

Comment:
Note that, for simplicity, this overwrites the `hashed_email` column if it is already populated.